### PR TITLE
safe cluster startup

### PR DESCRIPTION
**Pull request by kkraune**

### CODE REVIEW

File: en/config-sentinel.html [hunk 1]
```diff
@@ -1,16 +1,13 @@
 ---
-# Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 title: "Config sentinel"
 ---
 
 <p>
 The config sentinel starts and stops services -
-and restart failed services unless are manually stopped
+and restart failed services unless are manually stopped.
```
Comment (arnej27959) [inline en/config-sentinel.html:8]: old missing word: unless *they* are

File: en/config-sentinel.html [hunk 2]
```diff
@@ -109,3 +107,64 @@ <h2 id="user-interface">User interface</h2>
...
+</p>
+<p>
+  The safe mode is disabled by default,
+  enable it by setting configuration on top level in <em>services.xml</em> (since Vespa-7.415):
+</p>
+<pre>
+&lt;services&gt;
+    &lt;config name="cloud.config.sentinel"&gt;
+        &lt;connectivity&gt;
+            &lt;maxBadOutPercent&gt;60&lt;/maxBadOutPercent&gt;
```
Comment (arnej27959) [inline en/config-sentinel.html:128]: I'm considering changing the name of the config variables:
maxBadPercent (all problems count)
maxOneWayCount (affirmed missing connectivity one way)

**Review by arnej27959 — APPROVED** [review]
looks good, but it's likely some details will change before end of sprint, so maybe wait with merging?

File: en/config-sentinel.html (hunk 2 above)
Comment (kkraune) [reply en/config-sentinel.html:128]: yes, those are better names!

**Comment by kkraune** [comment]
yes lets merge when changes are made, converting to draft for now

**Comment by arnej27959** [comment]
the naming should be stable now, so i guess this PR can be merged now

**Comment by kkraune** [comment]
Thanks @arnej27959 - lets create a separate PR for the command to inspect status

**Pull request merged**